homeassistant/components/lamarzocco/services.py

```diff
@@ -0,0 +1,163 @@
+"""Services for the La Marzocco integration."""
+
+from collections.abc import Callable, Coroutine
+import logging
+from typing import Any
+
+from lmcloud.exceptions import AuthFail, RequestNotSuccessful
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry, ConfigEntryState
+from homeassistant.core import HomeAssistant, ServiceCall, callback
+from homeassistant.exceptions import HomeAssistantError, ServiceValidationError
+from homeassistant.helpers import config_validation as cv, selector
+
+from .const import (
+    CONF_CONFIG_ENTRY,
+    CONF_DAY_OF_WEEK,
+    CONF_ENABLE,
+    CONF_HOUR_OFF,
+    CONF_HOUR_ON,
+    CONF_MINUTE_OFF,
+    CONF_MINUTE_ON,
+    DAYS,
+    DOMAIN,
+    SERVICE_AUTO_ON_OFF_ENABLE,
+    SERVICE_AUTO_ON_OFF_TIMES,
+)
+from .coordinator import LaMarzoccoUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+
+CONFIG_ENTRY_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_CONFIG_ENTRY): selector.ConfigEntrySelector(
```

Comment:
Can the services be [entity services](https://developers.home-assistant.io/docs/dev_101_services/#entity-services) on the main entity instead?

IF yes there is no need for `__get_coordinator`?

Reply:
yes, but somehow I can call them in tests, but no code is executed, any idea? Also should I add it to the calendar platform PR, or separate PR?